internal/templates/commands/render_test.go

```diff
@@ -0,0 +1,217 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/abcxyz/pkg/testutil"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestParseFlags(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name    string
+		args    []string
+		want    flagValues
+		wantErr string
+	}{
+		{
+			name: "all flags present",
```

Comment:
https://github.com/abcxyz/readability/wiki/Go-Best-Practices#table-driven. This should be "all_flags_present" to that it matches the test name and can be targeted with `-run`. (Throughout all tests).

Reply:
Done.